chart/templates/statefulset.yaml

```diff
@@ -420,6 +420,25 @@ spec:
           ports:
             - containerPort: 9000
               name: yugaware
+          livenessProbe:
+            failureThreshold: 10
+            httpGet:
+              path: /api/app_version
+              port: yugaware
+              scheme: HTTP
+            periodSeconds: 30
```

Comment:
same here, there is a change on the yugaware helm chart that we can most likely copy from.